python/src/energuide/dwelling.py

```diff
@@ -0,0 +1,38 @@
+import typing
+
+
+class NoInputDataException(Exception):
+    pass
+
+
+class InvalidInputDataException(Exception):
+    pass
+
+
+DwellingData = typing.Iterable[typing.Dict[str, typing.Any]]
+
+
+class Dwelling:
+
+    def __init__(self, *, house_id: int) -> None:
+        self._house_id = house_id
+
+    @staticmethod
+    def verify_schema(data: DwellingData):
```

Comment:
Since it's not needed anywhere else, we can make it private

Reply:
Good catch